README.md

```diff
@@ -98,6 +98,19 @@ You can find the documentation for each module within their respective directori
 - [Swappable](src/Swappable)
   - [SwappableEvent](src/Swappable/SwappableEvent)
 
+## Running examples
+
+To run the examples locally run
+
+```
+$ yarn install
+$ yarn build
+$ yarn start
+```
+
+This will start a server that hosts the contents of `examples/`. It also watches for file
+changes from both `src/` and `examples/src` and reloads the browser.
+
```

Comment:
@ianmart1n This probably needs rephrasing ^ let me know if you have better alternatives. I'll merge the PR for now, but feel free to comment here or even submit a PR 👍